src/uct/ib/base/ib_iface.h

```diff
@@ -226,7 +226,8 @@ enum {
     /* Indicates that TX cq len in uct_ib_iface_init_attr_t is specified per
      * each IB path. Therefore IB interface constructor would need to multiply
      * TX CQ len by the number of IB paths (when it is properly initialized). */
-    UCT_IB_TX_OPS_PER_PATH           = UCS_BIT(2)
+    UCT_IB_TX_OPS_PER_PATH           = UCS_BIT(2),
+    UCT_IB_IS_GGA                    = UCS_BIT(3)
```

Comment:
I think it will be useful to have a comment here or in `uct_rc_mlx5_iface_common.gga`. It would be nice to have at least one place where it is indicated what the abbreviation stands for.

Reply:
I think need to remove this flag. instead have TM_ENABLE flag on mlx5 level